### ISSUE TITLE
For windows it can run in Debug mode but can't in Release mode

### ISSUE THREAD

**Issue by gangli66** [react-native-webview/react-native-webview#2588]
When I test my app in debug mode, it will run normally. However, when I produce release exe file and run, it will tell me I need to install 3 dlls like below:
msvcp140_app.dll
vcruntime140_app.dll
vcruntime140_1_app.dll
I had download them from https://www.dll-files.com/ but when I run the release exe again, there is no any message but don't disappear the windows that can disappear in debug mode.

**To Reproduce:**

**Expected behavior:**

**Screenshots/Videos:**

**Environment:**
 - OS: Win11
 - OS version: 21H2 22000.739
 - react-native version: 0.67
 - react-native-webview version: 11.22.4

**Comment by gangli66** [comment]
Thanks for your answer. I generated it with VS2019. I Seleted Release/x64, and then clicked the Run button. After that there was a directory named Release and including myapp.exe. When I double clicked the exe, it told me that I need three dll like above. I also generated appx by VS2019, but as a freshman, I don't know how to install and run it yet. Did I have any mistake to run that exe? Or I shouldn't use that exe, but need a appx? Thank you very much.
